tests/unit/python/foglamp/services/common/test_microservice.py

```diff
@@ -5,6 +5,7 @@
 from unittest.mock import patch, MagicMock
 from aiohttp import web
 import asyncio
+import sys
 from foglamp.common.storage_client.storage_client import ReadingsStorageClientAsync, StorageClientAsync
 from foglamp.common.process import FoglampProcess, SilentArgParse, ArgumentParserError
```

Comment:
Remove `SilentArgParse, ArgumentParserError` import

Reply:
Please remove as requested and if not requird